packages/babel-plugin-transform-destructuring/src/util.ts

```diff
@@ -637,6 +637,7 @@ export function convertVariableDeclaration(
   } else {
     path.replaceWithMultiple(nodesOut);
   }
+  path.scope.crawl();
```

Comment:
Q: Is Line 625 - 633 still required now that we crawl the scope?

Reply:
I don't understand the plugin source very well but removing them doesn't cause any tests to fail. Should I remove and commit then?